homeassistant/components/light/hue.py

```diff
@@ -235,19 +236,28 @@ def _color_mode(self):
     @property
     def hs_color(self):
         """Return the hs color value."""
-        # Don't return hue/sat if in color temperature mode
-        if self._color_mode == "ct":
-            return None
+        # pylint: disable=redefined-outer-name
+        mode = self._color_mode
 
-        if self.is_group:
-            return (
-                self.light.action.get('hue') / 65535 * 360,
-                self.light.action.get('sat') / 255 * 100,
-            )
-        return (
-            self.light.state.get('hue') / 65535 * 360,
-            self.light.state.get('sat') / 255 * 100,
-        )
+        if mode not in ('hs', 'xy'):
+            return
+
+        source = self.light.action if self.is_group else self.light.state
+
+        if mode == 'hs':
```

Comment:
You can remove this branch and the `get()` will work even in the HS case.

Reply:
Good point, made the code simpler 👍